scripts/apps/vocabularies/services/VocabularyService.ts

```diff
@@ -46,10 +46,11 @@ export function VocabularyService(api, $q, $filter, $rootScope) {
      * @name VocabularyService#getVocabularies
      * @public
      * @description Returns the manageable vocabularies.
+     * @param {String} vocabularyDeleted if true, fetch new vocabularies values again
      * @return {Promise} {Object} vocabularies
      */
-    this.getVocabularies = function() {
-        if (_.isNil(self.vocabularies)) {
+    this.getVocabularies = function(vocabularyDeleted) {
```

Comment:
`noCache` would be more clear as an argument name. It'd even better to use an options object so usages are more clear too e.g. `getVocabularies({noCache: true})`